tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -3534,11 +3534,6 @@ def test_ensure_case_when_no_metrics_index_not_found_is_handled_gracefully(self)
             }
         )
         self.assert_no_active_incident(rule)
-        self.entity_subscription_metrics.incr.assert_has_calls(
-            [
-                call("incidents.entity_subscription.metric_index_not_found"),
-            ]
-        )
```

Comment:
Shouldn't this still happen on V2?

Reply:
V2 doesn't go down the same code path - V1 called [this function](https://github.com/getsentry/sentry/blob/master/src/sentry/snuba/entity_subscription.py#L378) and the `MetricIndexNotFound` error was raised [here](https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/utils.py#L88-L96). Is that expected?

Reply:
This made me realize nothing calls `translate_sessions_tag_keys_and_values` anymore so I can remove that too.

Reply:
Yes, makes sense. Thanks for cleaning this up!